### DISCUSSION TITLE
Release 1.8.4

### DISCUSSION THREAD

**Discussion by jpsimen** [onnx/tensorflow-onnx#1405] [General]
Hi @guschmue,

Are you planning a new release within the next few days? I'm really looking forward to this commit: https://github.com/onnx/tensorflow-onnx/commit/e12d3eaf2e2cdf5dd04d77459294f5de9e726f67
If you are not planning to release it soon, I would clone the master branch instead of waiting for the pip package.

Thank you!
Jan

**Comment by guschmue** [comment]
We want to get one more fix in. Planning on today or tomorrow to push to pypi.

**Reply by guschmue** [reply]
pushed tf2onnx-1.8.4

**Reply by jpsimen** [reply]
That was quick. Thank you!
